### ISSUE TITLE
Feature Request: User defined column break

### ISSUE THREAD

**Issue by magicmasterp** [Secular12/cortexprime#71]
Is there a way to allow the user to define where the column break should be?
When I first made it my character sheet looked like this:
![Original Character Sheet](https://user-images.githubusercontent.com/110814800/183387996-d374f0dc-ddca-44ad-a8c4-8b56c3a574b4.jpg)

I switched the order of some of the traits and got something more balanced but then I noticed the column break didn't occur in the same place on all character sheets.
 
![New Layout Comp 1](https://user-images.githubusercontent.com/110814800/183389086-3c867056-146d-455d-9c3a-cce2b0f4cf22.jpg)  ![New Layout Comp 2](https://user-images.githubusercontent.com/110814800/183389596-01e07734-0bcb-40fb-bce9-aa93bc5ed2df.jpg)

**Comment by Secular12** [comment]
Yes, this is actually in my To Do list. The solution will be that when you add a trait set in the settings you will choose which column it gets added to and you can rearrange the order of each set in each column, and decided move them back and forth between columns.

**Comment by magicmasterp** [comment]
Awesome.

**Comment by olavmueller** [comment]
This would be very great. Can I send in my vote to up this onto the near top of your todo list? ;-)

**Comment by Secular12** [comment]
> This would be very great. Can I send in my vote to up this onto the near top of your todo list? ;-)

This *will* be in the next release, the next release, however, will be sometime I am completely overhauling the system. Changing traits to be items rather than just data in an Actor Sheet.
